index.js

```diff
@@ -229,7 +229,8 @@ async function installBundler(bundlerVersionInput, lockFile, platform, rubyPrefi
     console.log(`Using Bundler 1 shipped with ${engine}`)
   } else {
     const gem = path.join(rubyPrefix, 'bin', 'gem')
-    await exec.exec(gem, ['install', 'bundler', '-v', `~> ${bundlerVersion}`, '--no-document'])
+    const bundlerVersionConstraint = bundlerVersion.match(/^\d+\.\d+\.\d+$/) ? bundlerVersion : `~> ${bundlerVersion}`
```

Comment:
I think we should remove the `$` here so e.g. `2.2.0.rc.2` would also be considered an exact version.
(https://rubygems.org/gems/bundler/versions)

Reply:
@eregon Done!

Reply:
@eregon Done!